app/markets/Bet.tsx

```diff
@@ -60,6 +60,36 @@ export const Bet = ({ fixedProductMarketMaker }: BetProps) => {
       </div>
     );
 
+  if (marketModel.currentAnswer !== null && !marketModel.isAnswerResolved)
```

Comment:
Can we get a name for this condition? I think that would be better.

Reply:
ok, we can leave as it is for now, but wondering if it was an variable was a bit more clear.